jose-jwt/jose-jwt.csproj

```diff
@@ -117,6 +117,7 @@ RFC 7797:
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
+	<PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
why do we need it? .net46 should include built-in json support. 
My plan was actually to drop newtonsoft going forward as netcore 3 includes built-in too.

Reply:
Yes - ok... I've now removed that dependency for net461 from the main project and use only the wrapping class now not newtonsoft directly.